datahub-web-react/src/app/ingest/source/builder/sources.json

```diff
@@ -34,6 +34,13 @@
         "docsUrl": "https://datahubproject.io/docs/generated/ingestion/sources/looker/",
         "recipe": "source:\n    type: looker\n    config:\n        # Coordinates\n        base_url: # Your Looker instance URL, e.g. https://company.looker.com:19999\n\n        # Credentials\n        # Add secret in Secrets Tab with relevant names for each variable\n        client_id: \"${LOOKER_CLIENT_ID}\" # Your Looker client id, e.g. admin\n        client_secret: \"${LOOKER_CLIENT_SECRET}\" # Your Looker password, e.g. password_01"
     },
+    {
+        "urn": "urn:li:dataPlatform:lookml",
```

Comment:
How do we find the logo forthis guy?

Reply:
answered off of github - I have a mapping for this to looker which i have a mapping to get the logo from there